xflags-macros/src/parse.rs

```diff
@@ -156,6 +156,26 @@ fn ty(p: &mut Parser) -> Result<ast::Ty> {
 }
 
 fn opt_doc(p: &mut Parser) -> Result<Option<String>> {
+    let mut res = match opt_doc_line(p)? {
+        Some(doc) => doc,
+        None => return Ok(None),
+    };
+
+    loop {
```

Comment:
isn't this `while let Some`?

Reply:
This is :) . I guess lately I've been feeling that explicit match is more clear sometimes, but I'm happy to change it.

Reply:
Yeah, I’d prefer a while let